packages/turf-helpers/lib/geojson-equality.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  Feature,
+  LineString,
+  Position,
+  GeoJSON,
+  Point,
+  Polygon,
+  GeometryCollection,
+  FeatureCollection,
+  MultiLineString,
+  MultiPoint,
+  MultiPolygon,
+} from "geojson";
+import equal from "deep-equal";
```

Comment:
Did a test publish and when trying to use GeojsonEquality in a test script got the error "Error: Cannot find module 'deep-equal'". You might need to add deep-equal as a dependency in packages/turf-helpers/package.json

Reply:
Thanks for this, and apologies for not checking this first time around. I have added the dependency. In the process I have:

- Upgraded to deep-equal 2.2.3 (previous version was 1.1.1 which is 4 years old)
- Removed the deep-equal dependency from turf-line-overlap, since it depends on turf-helpers

I don't _think_ this should have any negative consequences for turf-line-overlap, but would be grateful if you can sanity-check that change.

Reply:
Good work getting deep-equal updated. Though as turf-line-overlap still uses deep-equal directly we should keep it as an explicit dependency in package.json. If you can reinstate that we should be good to go 👍

Reply:
Done